data-es5.js

```diff
@@ -67,8 +67,7 @@ exports.tests = [
       webkit: true,
       chrome5: null,
       chrome7: true,
-      opera10_50: null,
-      opera12_10: true,
```

Comment:
Sorry, do you want me to add "true" to all tested versions?  This would be a lot of work with little to no gain.